### PR TITLE
Add Custom scheme for example/sanitizers

### PR DESCRIPTION
**Pull request by chiragramani**
This PR updates the existing `examples/sanitizers` application with an addition of a custom scheme. The custom scheme attaches the default state applicable for respective sanitizer applications.

### CODE REVIEW

File: examples/sanitizers/BUILD [hunk 1]
```diff
@@ -3,14 +3,18 @@ load(
...
     "XCODEPROJ_TARGETS",
+    "get_xcode_schemes",
 )
 
 xcodeproj(
     name = "xcodeproj",
     build_mode = "bazel",
     project_name = "Sanitizers",
+    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
+    schemes = get_xcode_schemes(),
```
Comment (brentleyjones) [inline examples/sanitizers/BUILD:17]: You'll need to set this in `test/fixtures/BUILD` as well.

Comment (chiragramani) [reply examples/sanitizers/BUILD:17]: `examples` is present in the root `.bazelignore` and is being filtered out. What would be your suggestions to go about it?

Comment (brentleyjones) [reply examples/sanitizers/BUILD:17]: I mean you need to edit this: https://github.com/buildbuddy-io/rules_xcodeproj/blob/d316ac374b4f38d73d882647e0b36593cfe2fb32/examples/sanitizers/test/fixtures/BUILD#L10

Comment (chiragramani) [reply examples/sanitizers/BUILD:17]: Oh okay, makes sense! Updated.

**Review by brentleyjones — APPROVED** [review]

**Comment by chiragramani** [comment]
Looks like all the sanitizer related todos are merged. Thank you @brentleyjones for all your help in implementing this! 🙌🏼

**Pull request merged**